Tasks/Maslov/task2/Dz2.py

```diff
@@ -0,0 +1,43 @@
+
+import datetime
+
+a = datetime.datetime.now()
+print(a)
```

Comment:
you suppose to let user choose what time should be processed to a text version, not just print out the current time. read the task description ne more time please